SolutionStack/sysdig/adoc/TRD-Kubernetes-ss-sysdig-gs.adoc

```diff
@@ -0,0 +1,247 @@
+
+In this example use case, we will create a custom runtime security policy that will detect when a shell is spawned in a container with an attached terminal.  Commands being executed with a terminal inside a running container can be an indicator of a potential infiltration or attack.
+
+NOTE: For this example we are creating a *simple* custom rule that can be inhibited by other rules; thus we strongly recommend disallowing “Suspicious Container Activity” default rule before adding the new rule (Policies > Runtime Policies > Suspicious Container Activity).  After the experiment you may want to reactivate it again.
+
+
+=== Create the custom runtime security policy
+
+Log into Sysdig Secure and, in the Sysdig UI on the left panel, select Policies > Runtime Policies.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-select-runtime.png[scaledwidth="50%", align="center"]
+
+This opens the *Runtime Policies* page, where you can see the pre-defined policies.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-Runtime-Policies.png[scaledwidth="75%", align="center"]
+
+
+To create your own, custom policy, first select Add Policy in the upper right corner.  Then, in the pop-up screen, select Workload Policy.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-choose-policy.png[scaledwidth="75%", align="center"]
+
+In the New Policy screen, enter a name and a description:
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-new-policy-name.png[scaledwidth="75%", align="center"]
+
+Keep the policy defaults: Enabled with severity level of High.i  You can customize the scope of this rule, indicating where it should apply.  In this example, the scope is contained to a namespace, called terminal-shell-in-container.  You can also set this to a broader scope, such as kubernetes.cluster.name, for example.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-policy-scope.png[scaledwidth="75%", align="center"]
+
+Next, you’ll want to assign a rule to use with the policy.  Select Import from Library.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-import.png[scaledwidth="75%", align="center"]
+
+Find and select "Terminal shell in a container" from the Rules Library, then click Import Rules.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-rules-library.png[scaledwidth="75%", align="center"]
+
+You should now see the "Terminal shell in a container" rule assigned to your policy.
+
+image::TRD-Kubernetes-ss-sysdig-gs-image-rule-config.png[scaledwidth="75%", align="center"]
+
+
+Next, under Actions, you can choose whether you’d like to kill, stop, or pause the container when the policy is triggered.  For this example, leave this set to Nothing.  Read more about Actions https://docs.sysdig.com/en/docs/sysdig-secure/policies/manage-policies/#define-actions[here].
+
+Your email is the default notification channel, but you can configure additional options under settings > notification channels. For example, you can choose to send notifications to a Slack Channel.  You can also leave this blank.
```

Comment:
Is the lowercase for "settings > notification channels" intentional? Or should it be `__Settings__ > __Notification channels__`?

Reply:
Updated case